CRM/RcBase/Processor/Base.php

```diff
@@ -174,12 +174,18 @@ public static function validateInput(
         }
 
         if (!$valid) {
-            throw new CRM_Core_Exception(sprintf('%s is not type of: %s', $name, $type));
+            // If value is not an array print it also
+            if (is_array($value)) {
+                $msg = sprintf('%s is not type of: %s', $name, $type);
```

Comment:
What is the use-case here? The arrays could be logged also. For example the [var_export](https://www.php.net/manual/en/function.var-export.php) function returns the string representation of the array. It could be used for this.

Reply:
good point